### PR TITLE
Update SKILL.tsv (added latest KOR lines)

### PR DESCRIPTION
**Pull request by Sunaries**
Added the latest lines from the _kor file. The actual _kor file and the rest has to be updated by IMC.

@imcgames

GitHub Logo

Is there a way for the developer to add 2 new lines when using **{nl}{nl}**? As you can see in the image above, you see a wall of text. This is not very appealing. It would look much neater if would tell you this in the following format:

"_Able to equip [Cloth] armor.
When equipped with 3 or more pieces of [Cloth] armor, increases your maximum SP and magic damage taken is decreased by 5%._

_When equipped with 4 [Cloth] armor pieces, your Magic Defense increases and magic damage taken is decreased by an additional 10%._"

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -2443,24 +2443,24 @@ SKILL_20150317_002442	$Rush: Stun
 SKILL_20150317_002443	{memo X}Enemies pushed away by [Rush] are inflicted with [Stun] for 2 seconds at a 5% chance per attribute level.
 SKILL_20150317_002444	$Impaler: Double Damage
 SKILL_20150317_002445	The original monster hit by[Impaler] receives double damage.
-SKILL_20150317_002446	Dragging Death: Control ally trap
-SKILL_20150317_002447	[Dragging Death] can drag traps and statues installed by allies.
-SKILL_20150317_002448	Stead Charge: Additional damage against Large monsters
+SKILL_20150317_002446	$Dragging Death: Ally Trap Control
+SKILL_20150317_002447	$Allies can drag prepared traps and statues of [Dragging Death].
+SKILL_20150317_002448	$Steed Charge: Large-type Additional Damage
```
Comment (terrapin47) [inline SKILL.tsv]: "Additonal damage to Large-type"?

Saying it this way sounds like the damage is Large-type.

File: SKILL.tsv [hunk 2]
```diff
@@ -2443,24 +2443,24 @@ SKILL_20150317_002442	$Rush: Stun
...
-SKILL_20150317_002459	[Iron Hook] prevents large monsters from moving for 0.5 seconds per attribute level.
-SKILL_20150317_002460	Double Weapon Assault: Sub-Weapon damage
-SKILL_20150317_002461	Physical damage of the sub-weapon used in [Double Weapon Assault] increases by 10 per level.
-SKILL_20150317_002462	Jolly Roger: Pirate Flag Defense
-SKILL_20150317_002463	Pirate flag defense of [Jolly Roger] increases by 5 per level.
+SKILL_20150317_002459	$Prevents Large-type monsters from moving for 0.5 second per attribute level with [Iron Hook].
+SKILL_20150317_002460	Double Weapon Assault: Sub-weapon damage
+SKILL_20150317_002461	$Increases the physical damage of the sub-weapon used with [Double Weapon Assault] by 10 per level.
+SKILL_20150317_002462	$Jolly Roger: Pirate Flag Defense
+SKILL_20150317_002463	$Increases the pirate flag's defense of [Jolly Roger] by 5 per level.
```
Comment (terrapin47) [inline SKILL.tsv]: This reads a little strange.  Maybe change the "of" to "from"?

File: SKILL.tsv [hunk 3]
```diff
@@ -3187,38 +3187,38 @@ SKILL_20150401_003186	{memo X}Change to Testudo formation. All troops move in a
 SKILL_20150401_003187	{memo X}Attack - #{CaptionRatio}#%{nl}Defense increase as much as increased attack
 SKILL_20150401_003188	{memo X}Change direction of formation
 SKILL_20150401_003189	{memo X}The direction you face become front
-SKILL_20150401_003190	Attack speed increases while defense decreases per damage received.
+SKILL_20150401_003190	$Increases your attack speed and attack when you take damage, but decreases defense.
```
Comment (terrapin47) [inline SKILL.tsv]: Suggest:
"When you take damage, increases attack speed and physical attack while decreasing defense."

File: SKILL.tsv [hunk 4]
```diff
@@ -3187,38 +3187,38 @@ SKILL_20150401_003186	{memo X}Change to Testudo formation. All troops move in a
...
 SKILL_20150401_003191	{memo X}Attack + #{CaptionRatio}#% {nl}Defense - #{CaptionRatio}#%{nl}Max. repeat #{CaptionRatio2}# times{nl}Duration #{CaptionTime}# seconds
 SKILL_20150401_003192	$Temporarily renders damage to the body incapable of inflicting knock back, knock down or stiffness.
 SKILL_20150401_003193	{memo X}{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Use back of sword to smack down enemy.
 SKILL_20150401_003194	{memo X}Kill the enemy within set attack counts for additional EXP and rewards. But damage and Incapable of combate penalty also increases while the effects last.
-SKILL_20150401_003195	Attack limit #{CaptionRatio}# times{nl}EXP, Drop rate x2
-SKILL_20150401_003196	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Deeply stab the enemy that collapsed.
+SKILL_20150401_003195	$Attack Limit: #{CaptionRatio}#{nl}2x EXP & Drop Rate
+SKILL_20150401_003196	${#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Release a deep stab to the enemy with your stab, causing it to collapse.
```
Comment (terrapin47) [inline SKILL.tsv]: Isn't this the description for punish?  Punish is something that can only be used on enemies that have been knocked down.  So it should be something like:
"Deeply pierce a collapsed enemy."

File: SKILL.tsv [hunk 5]
```diff
@@ -3298,16 +3298,16 @@ SKILL_20150401_003297	{memo X}{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Shoot
...
 SKILL_20150401_003304	Create Guardian Saint magic circle. Target receives damage instead of skill caster.
 SKILL_20150401_003305	{memo X}Guardian Saint Duration #{CaptionTime}#seconds{nl}Magic Circle Duration #{CaptionRatio}# seconds
 SKILL_20150401_003306	{memo X}Attack #{SkillAtkAdd}#{nl}Defense + #{CaptionRatio}# {nl}Target 10{nl}Duration #{CaptionRatio2}# seconds{nl}Use 1 Holy water
 SKILL_20150401_003307	{memo X}Additional damage +#{CaptionRatio}#{nl}Attack #{CaptionRatio2}#times{nl}Duration 30seconds{nl}Use 1 Holy Powder
-SKILL_20150401_003308	Divine attack + #{CaptionRatio}#{nl}Additional damage +#{CaptionRatio2}#{nl}Duration #{CaptionTime}# seconds{nl}Use 1 Gislotis
-SKILL_20150401_003309	Block state of Incapable of Combate once.
-SKILL_20150401_003310	Recover HP #{CaptionRatio}#%{nl}Invincible for #{CaptionRatio2}# seconds{nl}Duration #{CaptionTime}# seconds
+SKILL_20150401_003308	$Holy Property Attack: +#{CaptionRatio}#{nl}Additional Damage: +#{CaptionRatio2}#{nl}Duration: #{CaptionTime}# seconds{nl}Consumes 1 Gyslotis
+SKILL_20150401_003309	$Prevents you from getting Incapable of Combat once.
```
Comment (terrapin47) [inline SKILL.tsv]: "getting" -> "becoming"

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
You can do so by adding a space between the {nl}'s. Instead of {nl}{nl}, use {nl} {nl}.

As for the PR, please remove the new lines from KR client

**Comment by Sunaries** [comment]
@imcgames Thanks for the explanation. It works! I've also removed the new KR files.

**Pull request closed without merging**